docs/en/remote-flow.rst

```diff
@@ -532,10 +532,10 @@ The following is a non-normative example of the response from the Relying Party
 .. code-block:: http
 
   HTTP/1.1 200 OK
-  Content-Type: application/json;charset=UTF-8
+  Content-Type: application/json
 
   {
-    "redirect_uri": "https://relying-party.example.org/cb#response_code=091535f699ea575c7937fa5f0f454aee"
+    "redirect_uri": "https://relying-party.example.org/cb?response_code=091535f699ea575c7937fa5f0f454aee"
```

Comment:
the reasn why it was a fragment is to allow the registration of the redirect_uris in the metadata and using fragments to allow randomization of the endpoints

Reply:
Please, see also https://github.com/node-fetch/node-fetch/issues/211

Reply:
openid4vp non-normative example uses the fragment
https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-6.2-22